solutions/prob6.py

```diff
@@ -0,0 +1,34 @@
+# https://projecteuler.net/problem=6
+# Sum square difference of first 100 natural numbers
+
+import math
+
+class sumSquareDiff(object):
+	
+	def __init__(self, number):
+        	self.number = number
+
+	def square(self, no):
+        	return no*no
+
+	def sum(self):
+		sum1=0
+		N = self.number
+        	for i in range(1,N+1):
+			sum1=sum1+i
```

Comment:
You can do sum1 += i?
I would prefer to use some other variable instead of `sum1`.